### PR TITLE
Create Pete - Hashs

### PR DESCRIPTION
**Pull request by Camp91783**

### CODE REVIEW

File: Pete - Hashs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+They both store a collection of different objects but arrays use integers as its index but a hash lets you use any object type
+
+How do you retrieve a particular value from a Hash?
+student = {name = Peter, eye color = green}
+student =["eye color"]  should return green
+
+How do you add information to a Hash?
+you can add it similar to an array but have to use {}. first you name the hash then assign the value to the key
+
+How would you perform an operation on every element inside a Hash?
```
Comment (andrewek) [inline Pete - Hashs]: > How would you perform an operation on every element inside a Hash?

As a hint, you'll use `.each`

**Comment by andrewek** [comment]
Great start. Your answers are mostly correct. A thing I'd love to see is a brief example for each answer.

**Comment by Camp91783** [comment]
I'll update this evening.   Thanks Andrew.  Have a good rest of your Tuesday 

Pete

**Comment by andrewek** [comment]
@Camp91783 When/as you update, follow this pattern:

1. Open up your workspace (on Cloud9) and make your changes
2. `git add -A` or `git add .` (either is fine)
3. `git commit -m "made revisions"`
4. `git push origin master`

**Pull request closed without merging**